### DISCUSSION TITLE
Compiling Wingpanel without Applications

### DISCUSSION THREAD

**Discussion by ghost** [elementary/wingpanel#383] [Q&A] [answered]
Hi,

Is there a possibility to compile the wingpanel switching off Applications feature?

Thank you.
Tom

**Comment by davidmhewitt** [comment, marked as answer]
Wingpanel is modular, each item in the bar is a separate component with its own Github repository and is installed via its own package.

The applications menu is one such component and the code can be found here: https://github.com/elementary/applications-menu/

The applications menu is installed via a package called `slingshot-launcher`, if that package is not installed, then there will be no applications menu in wingpanel.
